introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm Dan . . . trying to get a job so I can work remotely and still enjoy life!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.